client_end_user/shared/src/commonMain/kotlin/presentation/profile/ProfileScreen.kt

```diff
@@ -1,15 +1,155 @@
+                title(Resources.strings.username)
+                subTitle("@${state.user?.username}")
+                title(Resources.strings.address)
+                subTitle(state.user?.addresses?.firstOrNull()?.address ?: "")
+                title(Resources.strings.email)
+                subTitle(state.user?.email ?: "")
+            }
+            whiteCard {
+                BpTextField(
+                    modifier = Modifier.fillMaxWidth().padding(top = 24.dp),
+                    text = state.fullName,
+                    onValueChange = listener::onFullNameChanged,
+                    label = Resources.strings.fullName,
+                    keyboardType = KeyboardType.Text,
+                    errorMessage = state.fullNameErrorMsg,
+                    isError = state.isFullNameError,
+                )
+                BpTextField(
+                    modifier = Modifier.fillMaxWidth().padding(top = 24.dp),
+                    text = state.phoneNumber,
+                    onValueChange = listener::onPhoneNumberChanged,
+                    label = Resources.strings.mobileNumber,
+                    keyboardType = KeyboardType.Text,
+                    errorMessage = state.mobileNumberErrorMsg,
+                    isError = state.isPhoneNumberError,
+                )
+                BpButton(
+                    modifier = Modifier.fillMaxWidth().padding(top = 24.dp),
+                    title = Resources.strings.save,
+                    enabled = state.isButtonEnabled,
+                    onClick = {
+                        listener.onSaveProfileInfo()
+                    },
+                )
+            }
+            whiteCard {
+                Row(
+                    modifier = Modifier.clickable(
+                        indication = null,
+                        interactionSource = remember { MutableInteractionSource() },
+                    ) { listener.onLogout() },
+                    horizontalArrangement = Arrangement.spacedBy(8.dp),
+                ) {
+                    Icon(
+                        painter = painterResource(Resources.images.logout),
+                        contentDescription = Resources.strings.logout,
+                        tint = Theme.colors.primary,
+                    )
+                    Text(
+                        text = Resources.strings.logout,
+                        style = Theme.typography.title,
+                        color = Theme.colors.primary,
+                    )
+                }
+            }
+        }
+    }
+
+    @Composable
+    private fun whiteCard(content: @Composable () -> Unit) {
+        Column(
+            modifier = Modifier.fillMaxWidth().padding(16.dp)
+                .clip(shape = RoundedCornerShape(Theme.radius.medium))
+                .background(Theme.colors.surface)
+                .padding(16.dp)
+        ) {
+            content()
+        }
+    }
+
+    @Composable
+    private fun title(text: String) {
```

Comment:
composable functions start with Capitalize letters ,will be
"Title"
also we need to choose another descriptive name